core/src/main/kotlin/org/evomaster/core/problem/externalservice/HostnameResolutionAction.kt

```diff
@@ -0,0 +1,45 @@
+package org.evomaster.core.problem.externalservice
+
+import org.evomaster.core.search.EnvironmentAction
+import org.evomaster.core.search.Individual
+import org.evomaster.core.search.StructuralElement
+import org.evomaster.core.search.gene.Gene
+
+class HostnameResolutionAction(
+    val hostname: String,
+    val resolvedAddress: String,
+    val resolved: Boolean
+) : EnvironmentAction(listOf()) {
+
+    fun getRemoteHostname(): String { return hostname }
+
```

Comment:
add an `init{}` block to validate the input, eg not empty, and regex for `resolvedAddress`